amads/melody/similarity/melsim.py

```diff
@@ -3,10 +3,12 @@
 This wrapper seeks to allow the user to easily interface with the melsim package using
 the Score objects in AMADS.
 
-Melsim is a package for computing similarity between melodies. It is based on SIMILE, which was
-written by Daniel Müllensiefen and Klaus Frieler in 2003/2004. Melsim is used to compare two
-or more melodies pairwise across a range of similarity measures. Not all similarity measures
-are implemented in melsim, but the ones that are can be used in AMADS.
+Melsim is a package for computing similarity between melodies, and was written by
```

Comment:
Is it fair to say "was written by Sebastian"? Not sure I would read that from Klaus's email. Better to write "is being developed by Sebastian Silas and Klaus Frieler"?